packages/gatsby/src/commands/build.ts

```diff
@@ -14,6 +14,7 @@ import { initTracer, stopTracer } from "../utils/tracer"
 import db from "../db"
 import { store, readState } from "../redux"
 import * as appDataUtil from "../utils/app-data"
+import { flush } from "../utils/page-data"
```

Comment:
could we name this better. All of the other function names were updated to things like `writePageData`. `flush` is super generic and not clear

Reply:
Agreed, on it

Reply:
Aliased as `flushPendingPageDataWrites`